mita/util.py

```diff
@@ -235,7 +235,17 @@ def match_node_from_label(label, configured_nodes=None):
 def match_node_from_labels(labels, configured_nodes=None):
     """
     Given a list of labels, map them to a configured node type so that it can
-    be created. All the labels must exist in the configured node
+    be created. All the "asked for" labels must exist in the configured node,
+    which will cause any subset to be matched.
+
+    For example::
+
+        >>> labels = ['python', 'amd64']
+        >>> match_node_from_labels(
+                ['python'], {'node1': {'labels': ['python', 'amd64']}}
```

Comment:
I don't understand the 'labels = ['python', 'amd64'] here; the call to match_node_from_labels doesn't use a global, does it?